### PR TITLE
Fixed minor error in wording in Scene Organisation tutorial

### PR DESCRIPTION
**Pull request by Lunadyn**

### CODE REVIEW

File: tutorials/best_practices/scene_organization.rst [hunk 1]
```diff
@@ -254,7 +254,7 @@ Choosing a node tree structure
 So, a developer starts work on a game only to stop at the vast possibilities
 before them. They might know what they want to do, what systems they want to
 have, but *where* to put them all? Well, how one goes about making their game
-is always up to them. One can construct node trees in a myriad of ways.
+is always up to them. One can construct node trees in myriad ways.
```
Comment (akien-mga) [inline tutorials/best_practices/scene_organization.rst:257]: That change seems to be up for debate: https://www.dictionary.com/e/how-do-you-use-the-word-myriad/

Let's use `countless` instead then?

Comment (mhilbrunner) [inline tutorials/best_practices/scene_organization.rst:257]:
```suggestion
is always up to them. One can construct node trees in countless ways.
```

Comment (mhilbrunner) [reply tutorials/best_practices/scene_organization.rst]: Yeah, lets go with that.

**Comment by mhilbrunner** [comment]
Merged, thanks!

**Comment by Calinou** [comment]
Cherry-picked to the `3.3` and `stable` branches.

**Pull request merged**